man/systemd-journald.service.xml

```diff
@@ -287,7 +284,19 @@ systemd-tmpfiles --create --prefix /var/log/journal</programlisting>
         <filename>/var/log/journal</filename> is not available, or
         when <option>Storage=volatile</option> is set in the
         <citerefentry><refentrytitle>journald.conf</refentrytitle><manvolnum>5</manvolnum></citerefentry>
-        configuration file.</para></listitem>
+        configuration file.</para>
+
+	<para>When <filename>systemd-journald</filename> stops writing to a file,
```

Comment:
is "stop" really the best word here? it sounds so "temporary", but what you are describing here is "archival", i.e. the one-way last step we apply to such a file. Maybe find a stronger word for this... "ceases to"? "concludes to"?